cmd/minikube/cmd/start.go

```diff
@@ -87,8 +86,9 @@ var startCmd = &cobra.Command{
 
 // Set default value for host data and config dir
 var (
-	hostConfigDirectory = "/var/lib/minishift/openshift.local.config"
-	hostDataDirectory   = "/var/lib/minishift/hostdata"
+	hostConfigDirectory  = "/var/lib/minishift/openshift.local.config"
+	hostDataDirectory    = "/var/lib/minishift/hostdata"
+	hostVolumesDirectory = "/var/lib/minishift/openshift.local.volumes"
```

Comment:
So here we set the default, right? So we have knowledge where we things should be. Why can we not check the existence of this path and potentially create it. Also, this is a parameter the user can specify. If 'oc cluster up' does indeed not create the directory, if it is not the default, we would need the check anyways. We would need to retrieve the _hostVolumesDir_ flag value (which would be either the default we specify here or some user specified value) and make sure the directory exists. I don't see how this even can be handled in the ISO.

On that note, I would love to know why _cluster up_ would not create this directory if it is not the default. That in itself seems questionable.